opt-env-conf-test/test_resources/docs/sub-commands/man.txt

```diff
@@ -23,31 +21,10 @@ sub-commands (-h|--help)
   option: --config-file FILE_PATH
   env: CONFIG_FILE FILE_PATH
   
-  command without subcommands
-  command: top
-    name
-    option: --name NAME
-    env: NAME NAME
-    config:
-      name: # or null
-        <string>
-    
-  
-  command with subcommands
-  command: sub
-    A
-    command: a
-    
-    B
-    command: b
-    
-    C
-    command: c
-    
-    D
-    command: d
-    
-  
+
+.Sh COMMANDS
+  top		command without subcommands
+  sub		command with subcommands
```

Comment:
I think we definitely want to see _everything_ in the manpage.
Unless we want mutiple manpages, one per command?

Reply:
I think a single manpage is the more idiomatic way here? I don't use manpages that often but I think it's not unusual for manpages being exhaustive and therefore longer